spec/support/acceptance/command_shell.rb

```diff
@@ -0,0 +1,102 @@
+module Acceptance::CommandShell
```

Comment:
Instead of the copy/pasta here, is it not possible to just rename the existing file and reuse it for both meterpreter and command shells? 👀

Reply:
Or if we can't rename the existing one, can we DRY this out

Reply:
Working on a prerequisite [PR](https://github.com/rapid7/metasploit-framework/pull/19428) to do some refactors to make this possible 👍

Reply:
Got the changes from the mentioned PR pulled in now.